src/atoms/Heading.js

```diff
@@ -14,11 +14,6 @@ const Heading = styled.h1`
 const SmallHeading = Heading.extend`
 	font-size: ${props => props.theme.variables.headingSmallSize};
 	line-height: ${props => props.theme.variables.headingSmallLineHeight};
-
-	@media screen and (min-width: ${props => props.theme.variables.largeWidth}) {
```

Comment:
We are going to want to keep this media query. Larger screens command larger font sizes.